### ISSUE TITLE
Creating a new edge creates another useless edge

### ISSUE THREAD

**Issue by nihiyama** [bcakmakoglu/vue-flow#155]
# Title
Creating a new edge creates another useless edge

## Expected Behavior
Only created Edges are displayed.

## Current Behavior
In addition to the created Edge, another Edge will be created at (0, 0) position.

<img width="1166" alt="スクリーンショット 2022-05-29 12 55 14" src="https://user-images.githubusercontent.com/41140574/170851396-800b0079-3784-4785-a586-8db1cfd1d6cf.png">

## Possible Solution
In src/utils/graph.ts, return of add Edge function fix.
```typescript
export const addEdge = (edgeParams: Edge | Connection, elements: Elements, defaults?: DefaultEdgeOptions) => {
  if (!edgeParams.source || !edgeParams.target) {
    console.warn("[vueflow]: Can't create edge. An edge needs a source and a target.")
    return elements
  }

  let edge
  if (isEdge(edgeParams)) {
    edge = { ...edgeParams }
  } else {
    edge = {
      ...edgeParams,
      id: getEdgeId(edgeParams),
    } as Edge
  }
  edge = parseEdge(edge, defaults)
  if (connectionExists(edge, elements)) return elements
  elements.push(edge)
  return elements
  // return [...elements, edge]
}
```

**Comment by bcakmakoglu** [comment]
As I said in the PR: I would in general recommend not using `addEdge` and instead using `addEdges` provided by `useVueFlow`.

`addEdge` is just a "legacy" version of `addEdges` and will most likely be removed in the future as it a) skips creating an actual change for Vue Flow (i.e. no `EdgeChange` of Type `AddEdgeChange` is triggered) and b) it skips applying default edge options that have already been defined as an option/prop.

If you're using Vue Options API try using it like this:
```vue
<script>
import { VueFlow, useVueFlow } from '@braks/vue-flow'

const { addEdges } = useVueFlow({ id: 'options-api' })

export default defineComponent({
  components: { VueFlow },
  data() {
    return {
      elements: [
        {
          id: '1',
          label: 'Node 1',
          position: { x: 0, y: 0},
        }
      ]
    }
  },
  methods: {
    // regular event handler
    handleConnect: (params) => {
      addEdges([params])
    }
  },
})
</script>

<template>
  <VueFlow v-model="elements" id="options-api" @connect="handleConnect" />
</template>
```

That should work and be more future proof :)

Though I would generally recommend just switching to composition API - you'll be better off using composition in the long run.

**Comment by bcakmakoglu** [comment]
Fixed with #154

**Comment by nihiyama** [comment]
Thank you for your suggestion!  
I will use `addEdges`
